src/main/java/org/spongepowered/api/plugin/PluginManager.java

```diff
@@ -45,6 +48,17 @@
     Optional<PluginContainer> fromInstance(Object instance);
 
     /**
+     * Finds the {@link PluginContainer} for the specified plugin instance.
+     *
+     * @param instance Plugin instance
+     * @return Plugin container of instance
+     * @throws PluginNotFoundException if the instance could not be found
+     */
+    default PluginContainer find(Object instance) {
```

Comment:
I agree with Minecrell...the get would throw an exception and you can listen for that exception instead.